src/test/java/com/epam/qalab/DeliverToTest.java

```diff
@@ -0,0 +1,110 @@
+package com.epam.qalab;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeliverToTest {
+    WebDriver driver;
+    String country = "Bulgaria";
```

Comment:
this variable is only used in one test, so you can define it in this test (inside)

Reply:
done :)

Reply:
Sometimes I leave these kinds of variables (like Bulgaria) "at the top of the class" (as Kristina originally did) when for some reason I need to change its values and play around with the code.
It is easier to find such a variable when it is not "hidden" in some method.
I don't know if I am breaking some clean code rules by doing this...
Marat could also have moved the "exec" variable to the "setUp()" method. But it was probably worth leaving it as a class variable (in my subjective opinion), because its value depends on the operating system (we can't run the test until it matches our OS, it should be easy to find).
In this case I simply don't know :)